web/views/map/editor.tpl

```diff
@@ -17,6 +17,6 @@
   </br>
   <div id="buttons">
     <input type="button" id="cancel" class="btn btn-secondary" value="{{_("Cancel")}}"/>
-    <input type="button" id="validate" class="btn btn-primary" value="{{_("Valid")}}"/>
+    <input type="button" id="validate" class="btn btn-primary" value="{{_("Validate")}}"/>
```

Comment:
"Valid" is an adjective in English, so it cannot be used like this on a button which validates. I think it would be even better to use a different word, like "Save", but I don't want to imply that the change is saved straight away. Perhaps "Done" could work too.

Reply:
"Save" is an other step, "Done" would be better.